src/DB/db.js

```diff
@@ -0,0 +1,9 @@
+exports.getEvents = {
+  id: 1,
+  title: 'Community Challenge',
+  eventLocation: 'Sirikwa Hotel',
```

Comment:
`location` is a shorter and better alternative to `eventLocation`, I also think location should be an object containing `name`, `street`, `city`

Reply:
Should an event not have a short description? Some short info about what it's about. For instance:
```
title: 'End year party'
description:'Join us as we celebrate a year of growth and collaboration'
```